lib/index.d.ts

```diff
@@ -669,19 +669,6 @@ export namespace GoogleAdsTypes {
      */
     keywords?: string[];
 
-    /**
-     * An array of test device IDs to whitelist.
-     *
-     * If using an emulator, set the device ID to `EMULATOR`.
-     *
-     * ```js
-     * await Interstitial.request('ca-app-pub-3940256099942544/1033173712', {
-     *   testDevices: ['EMULATOR'],
-     * });
-     * ```
-     */
-    testDevices?: string[];
```

Comment:
@dylancom this is the only part I don't understand - it seems like it's performing the exact  same job and has the same typing, but if you change the name from testDevices to testDeviceIdentifiers it becomes a breaking change. Is that on purpose? Is it desired? :thinking:

Reply:
Yes so, now you set this option once instead of per each ad type (if you use banners/interstials etc in 1 app). At first I thought to leave the name intact, but then I thought about aligning it with the naming used in the sdk. (As they did the same name change there). Can easily keep the original name though if desired.

Reply:
"The testDeviceIdentifiers property applies to all ad requests, while the old testDevices property was set per-request."

Reply:
Sounds like a breaking change in behavior, probably best to go ahead and break the code. Let's test that semver and get a v2 ;-)

Serves as a good time to let you know my philosophy in general - copied from webview: 

> This project follows semantic versioning. We do not hesitate to release breaking changes but they will be in a major version.

Breaking change happens, no problem. Just bump that major. Vamos!